recipes/vtk/.gitattributes

```diff
@@ -0,0 +1,3 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+*.bash text=lf
+*.sh text=lf
```

Comment:
Understand. Could you maybe use a global gitattributes file?

My concern is that we really don't want to include this sort of stuff in a recipe. If we do decide we want this, it should really belong in the repo level not just this recipe. Also, we will want to add it to conda-smithy.

If you think this is generally useful, could you please add this to a new PR and move this to a top level directory so we can have this discussion separately?